File: apps/ios/PassAlarm/Domain/Entity/SkipException.swift
import Foundation

struct SkipException: Identifiable, Equatable, Sendable, Codable {
    var id: UUID
    var date: String // "YYYY-MM-DD"
    var reason: SkipReason
    var createdAt: Date

    enum SkipReason: String, Codable, Sendable {
        case manual = "MANUAL"
        case holiday = "HOLIDAY"
        case system = "SYSTEM"
    }
}

File: apps/android/app/src/main/kotlin/com/nogeass/passalarm/domain/model/SkipException.kt
package com.nogeass.passalarm.domain.model

data class SkipException(
    val id: Long = 0,
    val planId: Long = 0,
    val date: String = "",
    val reason: SkipReason = SkipReason.MANUAL,
    val createdAt: Long = System.currentTimeMillis()
)

enum class SkipReason {
    MANUAL, HOLIDAY, SYSTEM
}
